diff --git a/Dockerfile b/Dockerfile
index 0e822a7..4f20d1c 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -24,7 +24,9 @@ RUN git clone --single-branch --depth 1 --branch ${GSTREAMER_VERSION} \
 
 # fix meson build error
 RUN sed -i '1i if get_option('"'"'doc'"'"').disabled()\n  subdir_done()\nendif' \
-    subprojects/gst-plugins-good/docs/meson.build
+    subprojects/gst-plugins-good/docs/meson.build && \
+    sed -i '1i if get_option('"'"'doc'"'"').disabled()\n  subdir_done()\nendif' \
+    subprojects/gst-plugins-bad/docs/meson.build
 
 RUN meson setup builddir \
     --prefix=/usr/local \
@@ -38,7 +40,7 @@ RUN meson setup builddir \
     -Dgstreamer:tools=disabled \
     -Dbase=enabled \
     -Dgood=enabled \
-    -Dbad=disabled \
+    -Dbad=enabled \
     -Dugly=disabled \
     -Dlibav=disabled \
     -Dges=disabled \
@@ -59,7 +61,8 @@ RUN meson setup builddir \
     -Dgst-plugins-base:videoconvertscale=enabled \
     -Dgst-plugins-base:audioresample=enabled \
     -Dgst-plugins-base:opus=enabled \
-    -Dgst-plugins-good:vpx=enabled && \
+    -Dgst-plugins-good:vpx=enabled \
+    -Dgst-plugins-bad:nvcodec=enabled && \
     meson compile -C builddir && \
     meson install -C builddir && \
     find /usr/local/lib -name "*.so*" -exec strip {} \;
